presets/4.3/vtx/Japanese_Bandplans.txt

```diff
@@ -0,0 +1,41 @@
+#$ TITLE: VTX Japanese Bandplans
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS:  vtx, vtx table, Japan, Japanese, JP, JA, Bandplans
+#$ AUTHOR: ToshihiroMakuuchi
+#$ DESCRIPTION: ----------
+#$ DESCRIPTION: This VTX preset conforms to the Japanese band plans.
+#$ DESCRIPTION: However, it is not a perfect fit for all VTXs. Please use it only as a reference.
+#$ DESCRIPTION:
+#$ DESCRIPTION: -- Amateur Radio License in Japan --
```

Comment:
It is not clear from the description, if a user can put these VTX channels ONLY when he has Amateur license or everyone can use mentioned channels.

Reply:
In Japan, it is available only to holders of amateur radio licenses. This section will be rewritten to be clearer.